### DISCUSSION TITLE
How to ease the transition from `caret` to `mlr3` - looking for ressources

### DISCUSSION THREAD

**Discussion by serkor1** [mlr-org/mlr3#1050] [Q&A]
Hi,

Thank you for a great package. I have been using {caret} for a long while and are now transitioning towards {mlr3} - is there any resources on how to ease this transition? 

I am especially interested in evaluating and comparing multiple models! :smile: 

Best,

Serkan

**Comment by be-marc** [comment]
Hey,

I think our book is the best way to start https://mlr3book.mlr-org.com/. Chapter 3 is about evaluating and benchmarking.
